cli/build.gradle

```diff
@@ -1,6 +1,5 @@
 plugins {
     id 'org.jetbrains.kotlin.jvm'
-    id 'io.gitlab.arturbosch.detekt'
```

Comment:
Why is this plugin removed?

Reply:
I don't understand gradle. I assumed since this file has no references to it, this is unnecessary?

The top level `build.gradle` references `detekt`, and I assumed the `apply plugin` thing attempts to apply the plugin to all subprojects? If so, doesn't that mean this is unnecessary, since the top level project includes it?

FWIW, with this change, the `./gradlew cli:detekt` task still seems to work?

```
~/partiql-lang-kotlin $ ./gradlew cli:detekt --info
...
> Task :cli:detekt UP-TO-DATE
...
```

Reply:
I also don't understand gradle too well. You seem to be right regarding the top level references to `detekt`, which is applied to all the subprojects. Since the `detekt` plugin isn't referenced in the other subprojects, could you also remove the `detekt` plugin from
- [lang/build.gradle](https://github.com/partiql/partiql-lang-kotlin/blob/d8fc1691b7335adb4856924f5b113e4698c16d34/lang/build.gradle#L29)
- [pts/build.gradle](https://github.com/partiql/partiql-lang-kotlin/blob/d8fc1691b7335adb4856924f5b113e4698c16d34/pts/build.gradle#L19)
- [testscript/build.gradle](https://github.com/partiql/partiql-lang-kotlin/blob/d8fc1691b7335adb4856924f5b113e4698c16d34/testscript/build.gradle#L3)

Reply:
Okay, done, thanks.